sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/client/SentinelApiClient.java

```diff
@@ -169,14 +188,17 @@ private StringBuilder queryString(Map<String, String> params) {
         return queryStringBuilder;
     }
     
-    private HttpUriRequest postRequest(String url, Map<String, String> params) {
+    protected static HttpUriRequest postRequest(String url, Map<String, String> params, boolean supportEnhancedContentType) {
         HttpPost httpPost = new HttpPost(url);
         if (params != null && params.size() > 0) {
             List<NameValuePair> list = new ArrayList<>(params.size());
             for (Entry<String, String> entry : params.entrySet()) {
                 list.add(new BasicNameValuePair(entry.getKey(), entry.getValue()));
             }
             httpPost.setEntity(new UrlEncodedFormEntity(list, Consts.UTF_8));
+            if (!supportEnhancedContentType) {
```

Comment:
Shall we add some comments here?

Reply:
They are placed at the relevant function and referenced here now.